modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -21,10 +21,11 @@ class NotificationEmail
     }.freeze
     SUPPORTED_FORMS = TEMPLATE_IDS.keys
 
-    def initialize(form_data:, form_number:, confirmation_number:, notification_type: :confirmation, user: nil)
-      @form_data = form_data
-      @form_number = form_number
-      @confirmation_number = confirmation_number
+    def initialize(config, notification_type: :confirmation, user: nil)
+      @form_data = config[:form_data]
+      @form_number = config[:form_number]
+      @confirmation_number = config[:confirmation_number]
+      @lighthouse_updated_at = config[:lighthouse_updated_at]
       @notification_type = notification_type
```

Comment:
Given that this is only used in one spot, does it make more sense to set the instance variable for the one that uses it?

```ruby
@template_id = TEMPLATE_IDS[form_number][notification_type]
```

Reply:
The `send` method does a check for `form_number` before this line so I'd rather not make this change since that would complicate the initializer, I think.

Reply:
Fair enough!